restore_backups.sh

```diff
@@ -1,5 +1,28 @@
 #!/bin/bash
 
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "NAME"
+  echo "	restore_backup - Restore Kolibri and Basline backup"
+  echo
+  echo "SYNOPSIS"
+  echo "	./restore_backup [path /to /Kolibri backup] [path /to /Basline backup] ... [NO DB's]"
+  echo
+  echo "DESCRIPTION"
+  echo "	Drops and recreates both Kolibri and Basline databases."
+  echo "	The script terminates all active connections to both Kolibri and Basline dbs."
+  echo "	This disconnects any leaners who might be logged in or submitting tests"
+  echo ""
+  echo "NOTE"
+  echo "	With no dbs, the script drops and recreates empty Kolibri and Basline databases"
+  echo "	Always backup before runing the script"
+  echo 
+  echo "Examples"
+  echo
+  echo "	./restore_backup "
```

Comment:
This example is wrong. The function will not work unless both kolibri and baseline backups are supplied

Reply:
The first example is for the NOTE but was not too descriptive
 `  echo "NOTE"
  echo "	With no dbs, the script drops and recreates empty Kolibri and Basline databases"`

> This example is wrong. The function will not work unless both kolibri and baseline backups are supplied